public/index.html

```diff
@@ -1 +1 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Moramia</title></head><body><h1>Hello Push</h1><img src="05ea1a20cacf228dae8f2360be01079c.png" alt=""><p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ea rem, placeat assumenda facere voluptatem amet molestias tempora reiciendis quibusdam non, eaque dolorem natus labore ut, magni facilis itaque! Blanditiis, error?</p><script src="main.js"></script></body></html>
\ No newline at end of file
+<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Moramia</title></head><body><main class="products-wrapper"><article class="product"><img src="89643598ec10c062d188655168ef9da8.jpg" alt="Frutos secos"><div class="product-content"><div class="line"></div><p class="product-price">From $180</p><h4 class="product-title">Frutos secos</h4></div></article><article class="product"><img src="f46f95c28f17d1cbd0c9058f03bac68c.jpg" alt="Nueces"><div class="product-content"><div class="line"></div><p class="product-price">From $180</p><h4 class="product-title">Nueces</h4></div></article><article class="product"><img src="b5b487ad56069c1fce08b1e58fd28697.jpg" alt="Frutos secos"><div class="product-content"><div class="line"></div><p class="product-price">From $180</p><h4 class="product-title">Frutos secos</h4></div></article><article class="product"><img src="c503a2d89fcf18d053976a87f076a7c9.jpg" alt="Frutos secos"><div class="product-content"><div class="line"></div><p class="product-price">From $180</p><h4 class="product-title">Frutos secos</h4></div></article></main><script src="main.js"></script></body></html>
```

Comment:
all the `/public` files should be on the `.gitignore`

Reply:
I thought they were there :(
![image](https://user-images.githubusercontent.com/36536646/83693215-4303a080-a5bb-11ea-80b4-74d941f577d8.png)